config/example/shards/shard0.yaml

```diff
@@ -0,0 +1,21 @@
+storage:
```

Comment:
Forgot to remove?

Reply:
No. We have to specify full path to the config field

Reply:
I've meant a file, speaking about this comment https://github.com/TrueCloudLab/frostfs-node/pull/45#discussion_r1097738834

Reply:
Oh, you meant all new config files. Ok